source/less/blocks/page-main.less

```
@@ -0,0 +1,9 @@
+.page-main__description {
+  .font(24px;300;29px);
+  .size(280px;462px);
+  color: @white;
+  background-image: url("/img/first-screen-image.png");
```

```suggestion
  background-image: url("../img/first-screen-image.png");
```

Это относительный путь. Файл стилей находится в каталоге **css**, а изображения в **img**. Оба каталога лежат на одном уровне относительно корня сайта:
```
.
├── css/
├── img/
```

Если написать путь к изображению, как у вас, то браузер начнёт искать картинку в каталоге **css/img**, но в css каталога **img** нет. Двоеточие `..` указывает браузеру перейти на один уровень вверх и уже там искать каталог **img**.